src/dynamics/video_player/player/player.js

```diff
@@ -1286,45 +1294,58 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
                         ctx.clearRect(0, 0, canvas.width, canvas.height)
                         ctx.drawImage(video, 0, 0, canvas.width, canvas.height);
                         cb(canvas, ctx);
-                    }.bind(this), 200);
+                    }.bind(this), 250);
+                },
+                removeImgElements: function(imgElements) {
+                    imgElements.forEach(img => {
+                        if (img.parentNode)
+                            img.parentNode.removeChild(img);
+                    })
```

Comment:
```suggestion
                removeElement: function(element) {
                    if (element.parentNode) element.parentNode.removeChild(element);
                },
                removeElements: function(elements) {
                    elements.forEach(this.removeElement)
```

Reply:
closing this branch cause I had rebase issue